ToDoList/Data/SqlToDoRepo.cs

```diff
@@ -27,6 +28,17 @@ public IEnumerable<ToDoItem> GetAllItems()
         public void UpdateToDo(ToDoItem toDo)
         {
             //No code here
+            //_context.ToDoItems.Update(toDo);
```

Comment:
this whole UpdateToDo is not needed, check my other comments, just retrieve todo, change its text or completed in the corresponding action, then save. 
besides, this will never make completed true, it's always false as I see the default value in the dto

Reply:
again, is this method needed?